src/main/java/core/basesyntax/servise/impl/FileReaderImpl.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax.servise.impl;
+
+import core.basesyntax.model.OperationsList;
+import core.basesyntax.servise.FileReader;
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.IOException;
+
+public class FileReaderImpl implements FileReader {
+    private static final int HEAD_OF_INPUT = 0;
+
+    @Override
+    public void read(File file) {
+        try {
+            BufferedReader bufferedReader = new BufferedReader(new java.io.FileReader(file));
+            String operation = bufferedReader.readLine();
+            while (operation != null) {
+                OperationsList.list.add(operation);
+                operation = bufferedReader.readLine();
+            }
+            OperationsList.list.remove(HEAD_OF_INPUT);
```

Comment:
let's remove first line in the next service or in Main
we should be able to read from any file with this service